src/reports/tables/Aggregate5.jsx

```diff
@@ -1,53 +1,65 @@
-import React from 'react'
-import PropTypes from 'prop-types'
+import React from "react";
```

Comment:
Could you revert the extraneous single quote -> double quote and semi colon changes?

Reply:
@wpears  sorry I missed this I'll get it updated I think this was atoms formatter going rouge